spec/services/identity_linker_spec.rb

```diff
@@ -30,6 +30,8 @@
       rails_session_id = SecureRandom.hex
       nonce = SecureRandom.hex
       ial = 3
+      acr_values = 'http://idmanagement.gov/ns/assurance/aal/1'
+      vtr = 'C2.Pb'
```

Comment:
my expectation is that we allow arrays here? are these array columns or single columns